config/config.exs

```diff
@@ -8,7 +8,7 @@ config :git_hooks,
       tasks: [
         {:cmd, "mix clean"},
         {:cmd, "mix git_hooks.install"},
-        {:cmd, "mix hex.outdated --within-requirements"},
+        # {:cmd, "mix hex.outdated --within-requirements"},
```

Comment:
I think you forgot to checkout this file

Reply:
I want to remove this checks in pre_push . can't do git push without --no-verify

Reply:
in this case maybe in a separate pr, I agree this check is annoying